pkg/terminal/terminal.go

```diff
@@ -81,14 +100,20 @@ func New(client service.Client, conf *config.Config) *Term {
 		client.SetReturnValuesLoadConfig(&LongLoadConfig)
 	}
 
+	if conf.SourceListLineColor < ansiBlack ||
+	conf.SourceListLineColor > ansiBrWhite {
+		fmt.Fprintf(os.Stderr, "invalid source-list-line-color, defaulting to ansiBlue(34)\n")
```

Comment:
Remove this Fprintf, so that people who don't set the variable don't get an error message.